var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -48,8 +48,9 @@ class Sqlite(AutotoolsPackage):
     variant('dynamic_extensions', default=True, description='Support loadable extensions')
     variant('rtree', default=True, description='Build with Rtree module')
 
-    depends_on('readline')
-    depends_on('zlib')
+    if not self.spec.satisfies('platform=windows'):
```

Comment:
This is causing https://github.com/spack/spack/runs/7472966301?check_suite_focus=true

Reply:
(there is no `self` here)